tmt/steps/report/reportportal.py

```diff
@@ -318,7 +318,10 @@ def go(self) -> None:
         launch_attributes = self.construct_launch_attributes(suite_per_plan, attributes)
 
         launch_description = self.data.launch_description or self.step.plan.summary
-
+        # Checks whether Testing Farm provided link to artifacts
+        if os.getenv('TMT_REPORT_ARTIFACTS_URL'):
+            launch_description = ", ".join(
```

Comment:
1. f-string would be probably more readable, `launch_description = f"{launch_description}, {os.getenv('TMT_REPORT_ARTIFACTS_URL')}"`
2. `launch_description` might be `None` at this point, and I suspect that `"None, https://artifacts.fo.bar"` would not make a very nice launch description :) How about adding a test for `None` variant?

```python
if not launch_description:
    launch_description = os.getenv('TMT_REPORT_ARTIFACTS_URL')

else:
    launch_description = f"{launch_description}, {os.getenv('TMT_REPORT_ARTIFACTS_URL')}"
```

Reply:
wouldn't it be better suited for the url to be a launch attribute? maybe we can have it at both places?

Reply:
I would certainly be for that and it seems like an easy change. But if you don't think it makes sense that is fine by me also.

Reply:
(No preference on my side, I’m here nust for the Python stuff, no idea about RP integration)

Reply:
Ack, if it's not clickable, let's not add it to attributes.